POS/Integration/04_AssignUsers.md

```diff
@@ -1,65 +1,30 @@
 [!!Create a store in POS](./06_CreateStore.md)
+[!! Create user group](../../Core1Platform/AdministratingCore1/03_ManageGroups.md#create-user-group)
+[!! Add user to user group](../../Core1Platform/AdministratingCore1/03_ManageGroups.md#add-user-to-user-group)
```

Comment:
Links in related content beziehen sich auf Topic, nicht auf den konkreten Unterkapitel/Procedure, auf den man im Fließtext verweist. Also hier 
[!!Manage the user groups](../../Core1Platform/AdministratingCore1/03_ManageGroups.md)